config/webpack.config.dev.js

```diff
@@ -74,7 +74,7 @@ module.exports = merge({
         loader: 'string-replace',
         query: {
           search: '@@API',
-          replace: 'http://localhost:8000/data-research/consumer-complaints/api/v1/'
```

Comment:
I'm running the API as its own Django app separate from refresh as I was having trouble with flags / routing for a bit with the UI injection.

 `npm start` for the UI, `runserver` for the API to keep things flexible locally without touching CF-gov just yet.